modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/BlockListener.java

```diff
@@ -242,4 +242,16 @@ public void onFlow(BlockFromToEvent e){
             }
         }
     }
+
+    @EventHandler
+    public void onBlockForm(BlockFormEvent e) {
```

Comment:
After testing and grinding my server to a complete halt, this has a few problems.

BlockFormEvent is used for everything from snow and ice forming to mushrooms, fire spreading and more.

I'm not sure a good solution, but this will not work.

Reply:
The event itself is causing lag?

Reply:
My mistake on this, it appears that the level.dat for my world got corrupted and set the randomTickSpeed to 3 million instead of 30.  The point about the rest forming is still most likely a problem, so I'd sort to only affect ice blocks for now.

Reply:
Or atleast a limited list of blocks.

Reply:
Ok, I've reverted the changes to just ice.